ui/src/features/Editor/components/ParamsPanel/index.tsx

```diff
@@ -9,20 +9,19 @@ import { ParamEditor } from "./components";
 type Props = {
   selected?: Node;
   onParamsSubmit: (nodeId: string, data: any) => void;
+  onClose?: () => void;
 };
 
-const ParamsPanel: React.FC<Props> = ({ selected, onParamsSubmit }) => {
-  // This is a little hacky, but it works. We need to dispatch a click event to the react-flow__pane
-  // to unlock the node when user wants to close the right panel. - @KaWaite
+const ParamsPanel: React.FC<Props> = ({
```

Comment:
Removing the hacky stuff de-synchronizes the canvas and the app state. By using this hacky code, you are relying on react-flow to make the updates to the node to deselect.